src/components/Heading.js

```diff
@@ -11,9 +9,5 @@ const StyledHeading = styled.h1`
 `;
 
 export default function Heading({text}) {
```

Comment:
You can use the "built-in" `children` prop instead of `text` which is a prop you have created yourself.
```suggestion
export default function Heading({children}) {
```